csharp/src/Ice/Connection.cs

```diff
@@ -15,31 +15,48 @@
 
 namespace ZeroC.Ice
 {
-    public enum ACMClose
+    /// <summary>Specifies the close semantics for ACM (Active Connection Management).</summary>
+    public enum AcmClose
     {
+        /// <summary>Disables automatic connection closure.</summary>
```

Comment:
We should remove all these Close and Hearbeat enumerators prefix. They come from local Slice which like old C++ had unscoped enums until recent versions of Ice.

Reply:
Fixed

Reply:
Fixed